src/components/Wizard/WizardContainer/AboutCompany/AboutCompany.js

```diff
@@ -2,14 +2,35 @@ import React, { Component } from "react";
 import "./AboutCompany.css";
 
 export default class AboutCompany extends Component {
-
 	state = {};
 
+	componentWillMount() {
+		const containerState = this.props.containerState;
+
+		console.log(containerState, "=== container state");
+
+		if (containerState) {
+			if (
+				containerState.companyName ||
```

Comment:
you are right, i must've mixed something up. changed to cleaner 
if (containerState) {
     this.setState({
        companyName: containerState.companyName,
        email: containerState.email,
        website: containerState.website,
	});
}
P.S
in short, this piece of code, passes container state to aboutCompany component, in case user will decide to visit step one to change his name, he will be able to see his previous settings.